store/chat.js

```diff
@@ -501,68 +470,50 @@ export const actions = {
   },
 
   [$actions.loadSettings] ({ commit }) {
-    // Global chat
-    let global = null;
-    try {
-      global = localStorage.getItem( 'globalchat' );
-    } catch ( error ) {
-      logger ( 'No global chat option found.' );
-    }
-    commit( $mutations.setGlobal, global !== null ? global : false );
-
-    // Timestamps
-    try {
-      const showTimestamps = localStorage.getItem( 'showtimestamps' );
-      if ( showTimestamps !== null ) commit( $mutations.setTimestamps, showTimestamps );
-      else commit( $mutations.setTimestamps, true );
-    } catch ( error ) {
-      logger ( 'No showTimestamps option found.' );
-    }
-
-    // Ignore users
-    try {
-      const ignore = localStorage.getItem( 'useignore' );
-      if ( ignore !== null ) commit( $mutations.setUseIgnore, JSON.parse( ignore ) );
-      else commit( $mutations.setUseIgnore, true );
-    } catch ( error ) {
-      logger ( 'No ignore option found.' );
-    }
-
-    // Notifications
-    try {
-      const notify = localStorage.getItem( 'notify' );
-      if ( notify !== null ) commit( $mutations.setNotify, notify );
-      else commit( $mutations.setNotify, true );
-    } catch ( error ) {
-      logger ( 'No notification sound option found.' );
-    }
-
-    // Autocomplete
-    try {
-      const autocomplete = localStorage.getItem( 'autocomplete' );
-      if ( autocomplete !== null ) commit( $mutations.setAutocomplete, autocomplete );
-      else commit( $mutations.setAutocomplete, true );
-    } catch ( error ) {
-      logger ( 'No autocomplete option found.' );
-    }
-
-    // Autocomplete
-    try {
-      const atInLocal = localStorage.getItem( 'at-in-local' );
-      if ( atInLocal !== null ) commit( $mutations.setRecieveMentionsInLocal, atInLocal );
-      else commit( $mutations.setRecieveMentionsInLocal, false );
-    } catch ( error ) {
-      logger ( 'No autocomplete option found.' );
-    }
-
-    // High density
-    try {
-      const useHighDensity = localStorage.getItem( 'high-density' );
-      if ( useHighDensity !== null ) commit( $mutations.setHighDensity, useHighDensity );
-      else commit( $mutations.setHighDensity, false );
-    } catch ( error ) {
-      logger ( 'No high-density option found.' );
-    }
+    let settings = new Map([
+      [$states.timestamps, $mutations.setTimestamps],
+      [$states.global, $mutations.setGlobal],
+      [$states.useTts, $mutations.setUseTts],
+      [$states.useIgnore, $mutations.setUseIgnore],
+      //TODO: fix ignore list
+      //[$states.ignoreList, $mutations.setIgnoreList],
```

Comment:
So does this break the ignore system or is this a "migrate to new style" later type deal?

Reply:
It's migrate to new style. 

The ignore list, from what I've seen with `git grep`, is, and always has been, completely separate in how it's saved/works, and migrating it would have been one giant commit spanning a lot of files (just calling `migrate()` wouldn't have cut it). 

This + a lot of other variables and subsystems living in no-man's-land will get refactored in subsequent rounds of refactoring.